packages/rrweb/src/record/observer.ts

```diff
@@ -93,7 +91,7 @@ export function initMutationObserver(
   // see mutation.ts for details
   mutationBuffer.init(options);
   let mutationObserverCtor =
```

Comment:
No idea what all the angularZoneSymbol stuff is, but now that this is no longer reassigned by that code, eslint requires us to change to a `const`

```suggestion
  const mutationObserverCtor =
```